examples/fasthtml/vercel.json

```diff
@@ -0,0 +1,12 @@
+{
+  "routes": [
```

Comment:
This is temporary and will be removed in a follow-up PR, but this solves a chicken + egg problem because the `fasthtml` preset is not yet accepted by our API endpoints.